app/models/concerns/digital_object/datacite.rb

```diff
@@ -103,15 +90,15 @@ def update_doi_metadata(target_url = nil)
   # Uses the modify identifier API call, returns true if metadata update was successful
   # if not, returns false
```

Comment:
Does this method ever return false anymore? Or does the comment need to be updated?

Reply:
Comment updated as suggested.